aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +261,40 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Forms in layers vs. full page 
+
+
+- Single column form layout creates a clear focused path to completion for the user.
+- Multi-column layouts can cause users to skip important fields, input data in a wrong field, or simply hinder efficiency which leads to frustration or even abandonment. 
```

Comment:
```suggestion
- Single column form layout creates a clear focused path to completion for the user.
- Multi-column layouts can cause users to skip important fields, input data in a wrong field, or simply hinder efficiency which leads to frustration or even abandonment. 
```

These are repeated below. They seem to flow well within the "avoid multi columns"

Reply:
done my mistake